docs/command/mongocli-atlas-privateEndpoints-onlineArchive-aws-create.txt

```diff
@@ -0,0 +1,77 @@
+.. _mongocli-atlas-privateEndpoints-onlineArchive-aws-create:
+
+========================================================
+mongocli atlas privateEndpoints onlineArchive aws create
+========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Create a new Data Lake private endpoint for your project.
+
+When you run this command:
```

Comment:
@andreaangiolillo I thought we are supposed to add annotations to .go files only and these txt files get autogenerated by the build. Where is this change in the go files? WAs there a different PR? Or was this stuff there from before?

Reply:
In this PR we are creating the command `onlineArchive` under `privateEnpoints` and then we are adding all the `privateEndpoint datalake` commands to `privateEnpoint onlineArchive`. So basically, we are reusing the same documentation of  `privateEndpoint datalake` for `privateEnpoint onlineArchive`